artifacts/client-with-secrets.yaml

```diff
@@ -4,7 +4,7 @@
 
 ## It uses secrets that are mounted into the Pod for the token and the license
 
-# kubectl create secret generic inlets-server-token --from-literal inlets-server-token=$TOKEN
+# kubectl create secret generic inlets-access-key --from-literal inlets-access-key=$TOKEN
```

Comment:
access-key is for cloud provisioning

so we should put "inlets-server-token" everywhere else IMHO